libs/domains/organizations/feature/src/lib/git-provider-setting/git-provider-setting.tsx

```diff
@@ -31,6 +31,22 @@ export const getGitTokenValue = (value: string) => {
   return null
 }
 
+export const getGitProvider = (url: string) => {
```

Comment:
We can turn that into a `util` lib

Reply:
https://github.com/Qovery/console/pull/1024/commits/46890c1023b8185f57e35db645522109c392f662